server/app/controllers/admin/BadRequestException.java

```diff
@@ -0,0 +1,12 @@
+package controllers.admin;
+
+/**
+ * Thrown when a request is malformed. This class is final to ensure that it works with {@link
+ * controllers.ErrorHandler} and subclasses aren't thrown with the same expected behavior.
+ */
+public final class BadRequestException extends RuntimeException {
```

Comment:
suggestion (blocking): how about moving https://github.com/seattle-uat/civiform/blob/main/server/app/controllers/api/BadApiRequestException.java#L8-L13 out of the api package and using it here as well?